### ISSUE TITLE
On 5.x decorated services cause TS error

### ISSUE THREAD

**Issue by mfeckie** [ember-decorators/ember-decorators#378]
When using the `service` decorator in 5.x like this

```ts
import { inject as service } from '@ember-decorators/service';

export default class Foo extends Component {
  @service foo
}
```

TS generates this error `[ts] Unable to resolve signature of property decorator when called as an expression. [1240]`

If, however, we do

```ts
import { inject as service } from '@ember-decorators/service';

export default class Foo extends Component {
  @service() foo
}
```

The compiler is happy.

Not sure if this is a documentation tweak needed or an actual bug

**Comment by pzuraq** [comment]
This is definitely a bug, we had tests for the JS behavior but not for the types. Got a PR out for it.

**Comment by pzuraq** [comment]
This should be fixed in `v5.0.1`
